src/Libraries/DSIronPython/IronPythonEvaluator.cs

```diff
@@ -32,7 +33,7 @@ public enum EvaluationState { Begin, Success, Failed }
     ///     Evaluates a Python script in the Dynamo context.
     /// </summary>
     [IsVisibleInDynamoLibrary(false)]
-    public static class IronPythonEvaluator
+    public class IronPythonEvaluator : Dynamo.PythonServices.PythonEngine
```

Comment:
API break?

Reply:
Should not be an API break...since all the public methods were static already